include/enrico/nek_driver.h

```diff
@@ -77,23 +76,12 @@ class NekDriver : public Driver {
   //!
   //! \param local_elem A local element ID
   //! \return The volume-averaged temperature of the element
-  double get_local_elem_temperature(int local_elem) const;
-
-  //! Return true if a global element is in a given MPI rank
-  //! \param A global element ID
-  //! \param An MPI rank
-  //! \return True if the global element ID is in the given rank
-  bool global_elem_is_in_rank(int global_elem) const;
-
-  //! Return true if a global element is in the fluid region
-  //! \param global_elem  A global element ID
-  //! \return 1 if the global element is in fluid; 0 otherwise
-  int global_elem_is_in_fluid(int global_elem) const;
+  double temperature(int local_elem) const;
```

Comment:
At one point, we discussed naming these functions:
``` cpp
double temperature_at(int local_elem) const;
```
To more-clearly distinguish them from
``` cpp
xt::xtensor<double, 1> temperature() const final;
```
What do you think?

Reply:
I'm down for that. It would help to make it a bit clearer.

Reply:
We could also define it as something like:
``` cpp
xt::tensor<doulbe, 1> temperature_at(const std:vector<int> &idx)
```

If so, we could include it in the `HeatFluidsDriver` base class.  

My reasoning is that `SurrogateHeatDriver` and `NekDriver` have different conventions for indexing regions, but this declaration could handle both.  You'd have an indexing scheme like `temperature_at({i,})` for `NekDriver` and `temperature_at({i, j, k})` for `SurrogateHeatDriver`

Reply:
Doesn't seem like that's really needed as part of the heat-fluids driver interface though. `temperature()` is sufficient for that. I'm ok just having those be specific to the individual drivers at this point.